go/vt/vtctl/reparentutil/durability_funcs_test.go

```diff
@@ -313,3 +313,70 @@ func Test_haveRevoked(t *testing.T) {
 		})
 	}
 }
+
+func Test_canEstablishForTablet(t *testing.T) {
```

Comment:
Unfortunately no, all the tests that set the durability policy cannot run in parallel since the durabilityPoilicy is stored as a global variable, so they conflict with each other